coderd/healthcheck/derphealth/derp_test.go

```diff
@@ -125,7 +127,7 @@ func TestDERP(t *testing.T) {
 		}
 	})
 
-	t.Run("ForceWebsockets", func(t *testing.T) {
+	t.Run("FailoverToWebsockets", func(t *testing.T) {
```

Comment:
"ForceWebsockets" is a setting that makes it not even attempt regular DERP upgrade.  Here we try Upgrade: DERP, get rejected, and then failover to websockets.